storage/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerDbManager.java

```diff
@@ -311,14 +492,23 @@ public void loadSampleServiceInfo(DataSource ds, UUID guid, Path sviValues, Path
       return;
     }
     try (Stream<Path> files = Files.walk(sviValues, 1)) {
-      files.map(Path::toAbsolutePath).filter(Files::isRegularFile).forEach((path) -> {
-        try {
-          byte[] value = Files.readAllBytes(path);
-          addServiceInfo(ds, path.getFileName().toString(), value, false);
-        } catch (IOException e) {
-          throw new RuntimeException(e);
-        }
-      });
+      files
+          .map(Path::toAbsolutePath)
+          .filter(Files::isRegularFile)
+          .forEach(
+              (path) -> {
+                try {
+                  if (!path.getFileName().toString().equals("activate_mod")) {
+                    byte[] value = Files.readAllBytes(path);
+                    addServiceInfo(ds, path.getFileName().toString(), value, false);
+                  } else {
+                    addServiceInfo(
+                        ds, path.getFileName().toString(), Composite.decodeHex("F5"), true);
```

Comment:
Shouldn't this be reading the file 'activate_mod', and pushing the subsequent CBOR value, instead of default 'True' here?
Also, since the keyword is used quite a lot, make it a constant

Reply:
Since we are removing the support for CBOR encoded types, this will change accordingly and then the value will be read from the file directly.

Reply:
I assume the "F5" relates to a cbor object. A comment indicating such would be useful.

Reply:
As part of PR 134  we had removed support for CBOR encoded value. When this PR was raised the CBOR encoded value was still part of the code, so you are seeing this piece of code in an outdated snapshot. If you go to the file in this PR with the latest commit, this piece of code referring to 'F5' has been removed.